command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
whether *we* define any other API or not isn't the point. People can develop any API/interface they want for their runtime - that's out of scope for us - but that doesn't mean they can't be compliant to the runtime. All they need to be able to do is adhere to the MUSTs in the runtime spec.

When you have a set of specs people will often want to support a subset of them as it fits their needs. We don't want to force someone to either be non-compliant or to support something they don't want/need.